infra/azure/terraform/secrets-store-csi-driver/main.tf

```diff
@@ -0,0 +1,74 @@
+/*
+Copyright 2024 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+provider "azurerm" {
+  features {}
+}
+
+# Data source to get the current client configuration
+data "azurerm_client_config" "current" {}
+
+# TODO: add state maintainence in Azure
```

Comment:
is this a note for the Azure remote backend?

Reply:
Yep